### ISSUE TITLE
Glutamate uncaging

### ISSUE THREAD

**Issue by ElanHR** [openconnectome/Statistical-Connectomics-Class#200]
This term keeps being used and I'm a little unsure of what is meant by it specifically. I'm been taking it to mean "exciting/lighting up (hopefully) the neuron of interest".

**Comment by imichaelnorris** [comment]
Someone who understand neuroscience better than I can should follow up with
a better explanation, but glutamate is a neurotransmitter involved with
neural activation. Uncaging I believe is referring to it being stimulated
by light through some experimental process, which lets us see that it's
been excited.

Read the introduction and at least the first paragraph of the methods
section.

http://en.m.wikipedia.org/wiki/Photostimulation

**Comment by yaxigeigei** [comment]
We know how to stimulate neurons electrically - simply shocking them with electrode. But how can we stimulate them using light which would be very convenient when doing imaging? As imichaelnorris said, glutamate is a kind of neurotransmitter naturally used by neurons to excite other neurons. However, we can't pour glutamate directly into the brain because ALL neuron will be activated ALL the time. Uncaging technique utilizes a modified version of glutamate, in this paper it is called Rubi-glutamate, which only becomes effective(excitatory) when activated by focal laser(uncaging, the "cage" is that Rubi-).

**Comment by ElanHR** [comment]
Ahh got it, that makes sense. Thanks!

**Comment by indivaux** [comment]
Bathing all the neurons with glutamate would also kill them, which is kind of bad... Luckily we have this great way of releasing as much as we want, when we want it, which is a much better option.
Basically: save neuron lives, use caged glutamate!

**Comment by whock** [comment]
Right, so you release a modified form of glutamate that is made inactive by a chemical group bound to it (the "cage") which physically blocks it from binding to any cells or anything. So Glut is there, resting, waiting. And so when the experimenter shines a specific wavelength of light, the link is broken (the "cage" is opened) and glutamate is then free to bind to cells. But the key part is that this happens locally - the laser only excites a small volume of glutamate. So it's a good way to chemically excite individual synapses. 

Here's a video of glutamate uncaging at individual synapses. You can clearly see the transient increase in [Glu]. Pretty cool. 

https://www.youtube.com/watch?v=dY7qKpi2u_c

**Comment by akim1** [comment]
Also, you can shine light at a very specific part of the brain to excite only the cells in that region.

**Comment by yaxigeigei** [comment]
Well, I don't think the video directly shows the transient increase of glutamate. Instead, the neuron was labeled by intracellular green calcium sensor and the fluorescent intensity was increased focally when that spine was activated by uncaged glutamate (=> depolarization => calcium influx => greener).

**Comment by akim1** [comment]
The caption in the video says that the green is GFP-GammaActin. This is just the rearranagement/formation of spines as a result of glutamate stimulation.

**Comment by yaxigeigei** [comment]
Sorry, I should read the video info first...
